templates/html/templates/common/macros.html

```diff
@@ -76,19 +76,29 @@
                         </a>
                     </div>
 
-                    <div class="collapse">
+                    <div class="collapse {{ accordion_content_state }}">
                         <div class="card-body">
+                        {% if code_mode %}
+                            <pre>
                             {% for idx, item in values.iteritems() %}
-                                <p>{{ item|e|replace("\n","<br/>") }}</p>
+                                <code property="{{ rdfa_property }}">{{ item|e|replace("\n","<br/>") }}</code>
```

Comment:
remember that we will have to define somewhere in the HTML header all the prefix definitions (skos:, sh:, dct:. owl: etc. ).